programs/fileio.c

```diff
@@ -862,6 +862,193 @@ int FIO_compressFilename(const char* dstFileName, const char* srcFileName,
     return result;
 }
 
+typedef struct {
+    int numActualFrames;
+    int numSkippableFrames;
+    unsigned long long decompressedSize;
+    int canComputeDecompSize;
+    unsigned long long compressedSize;
+    int usesCheck;
+} fileInfo_t;
+
+
+int calcFrameHeaderSize(BYTE frameHeaderDescriptor){
```

Comment:
`static` if the function is not meant to be called outside of its module.
It reduces number of entry points to protect.